server/src/test/java/io/crate/http/HttpTestServer.java

```diff
@@ -124,8 +127,6 @@ public void shutDown() {
     @ChannelHandler.Sharable
     public class HttpTestServerHandler extends SimpleChannelInboundHandler<Object> {
 
-        private final Logger logger = LogManager.getLogger(HttpTestServerHandler.class.getName());
-
```

Comment:
I removed logger as we can always debug - didn't want to overcomplicate and make it smth like `TriConsumer` only because of logger.